### ISSUE TITLE
Tooltip button focus order differs from visual order

### ISSUE THREAD

**Issue by henrycatalinismith** [zetkin/call.zetk.in#273]
## Steps to reproduce

1. Open an assignment.
2. Press `⌘ + F5` to start VoiceOver.
3. Focus the tooltip's `I understand` button.

## Expected outcome

The `I understand` button is visually first so it should be first in the focus order. See [C27: Making the DOM order match the visual order](https://www.w3.org/TR/WCAG20-TECHS/C27.html) for details of the success criteria this addresses and the assistive technology use cases it supports.

## Actual outcome

The link to the manual is first in the focus order despite being second visually.

![Screen recording demonstrating the focus order described above for the benefit of sighted people unfamiliar with keyboard navigation](https://user-images.githubusercontent.com/566159/191824489-9bf2a494-6053-4707-9f4a-31cc9f5372c0.gif)

## Severity

The success criterion covering this issue is WCAG level A – the minimum level. Most organizations aim for a conformance level of AA, which includes all level A criteria. In this particular case this issue seems unlikely to seriously hamper anybody's work but as it's level A it's still moderately severe.

## Recommendation

Rearrange the rendering of these two elements so that the DOM order matches the visual order.
